scripts/build/TestImpactAnalysis/storage_query_tool/local_storage_query_tool.py

```diff
@@ -124,13 +133,23 @@ def _update(self, file: str, storage_location: str):
         @param file: File to store.
         @param storage_location: Location to store the file.
         """
-        try:
-            json_obj = json.loads(file)
-            if self._check_object_exists(storage_location):
-                with open(f"{storage_location}", "w", encoding="UTF-8") as raw_output_file:
-                    json.dump(json_obj, raw_output_file,
-                              ensure_ascii=False, indent=4)
-            else:
-                logger.info("Cancelling update, as file does not exist")
-        except json.JSONDecodeError:
-            logger.error("The historic data does not contain valid json")
+        if self._file_type == self.FileType.JSON:
```

Comment:
Same here (see prev comment).

Reply:
Extracted out some of the code but left the control structure in this method.